Source/Request.swift

```diff
@@ -57,25 +57,23 @@ public protocol RequestRetrier {
 
 // MARK: -
 
+protocol TaskConvertible {
```

Comment:
Just double checking to see if it's necessary for this to be public? Perhaps if user's subclass everything they'd need this?